### ISSUE TITLE
The page where you see the entire message freezes if I send a wordlist 

### ISSUE THREAD

**Issue by leonardoporpora** [signalapp/Signal-iOS#3217]
- [x] I have searched open and closed issues for duplicates

----------------------------------------

### Bug description
The page where I see all the long message (tap for more) Signal freezes if I send a word list

### Steps to reproduce
- send a long wordlist
- then swipe to the left or click on the message (Tap For More)
- That page will be impossible to read because it freezes the page NOT the entire app

**Actual result:** Cannot read the entire message
**Expected result:** Should be able to read the entire message
### Screenshots
Not needed

### Device info

**Device**: iPhone 6s
**iOS version**: 11.3
**Signal version:** 2.23.0

### Link to debug log
https://debuglogs.org/97b1a1acf6ad8dc6ec547a6daf61c9e3c6a6db0d9b0bd80e0257abe3efad76cc.zip

<!--  If this is a crashing bug, after filing this issue, email a copy of your latest crash report to support@whispersystems.org

To get a crash log:

Since it doesn't freeze all the App I can't get a crash log because it doesn't crash

**Comment by leonardoporpora** [comment]
If the wordlist is very very very long also the Screen Security feature will have a high delay before be working

**Comment by michaelkirk-signal** [comment]
Thanks @leonardoporpora 

For the record I'm able to reproduce this with the text:
```
0
1
2
3
4
5
6
7
8
9
10
1
2
3
4
5
6
7
8
9
20
1
2
3
4
5
6
7
8
9
30
```

Rendered as:
<img width="375" alt="chopped" src="https://user-images.githubusercontent.com/36971200/38261661-e13e4cc6-3738-11e8-807b-592327961ab4.png">

**Comment by michaelkirk-signal** [comment]
This is fixed in the 2.24.* release.
